tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -140,65 +146,61 @@ More Information on raster metadata:
 * <a href="{{ site.baseurl }}/hyper-spec-intro" target="_blank"> Metadata and important concepts associated with multi-band (multi and hyperspectral) rasters </a>.
 
 <div id="ds-dataTip" markdown="1">
-<i class="fa fa-star"></i>**Data Tip - HDF5 Structure:** Note that the structure 
-of individual HDF5 files may vary depending on who produced the data. In this 
-case, the Wavelength and reflectance data within the file are both datasets. 
-However the spatial information is contained within a group. Data downloaded from 
-another organization like NASA, may look different. This is why it's important to 
+<i class="fa fa-star"></i>**Data Tip - HDF5 Structure:** Note that the structure
+of individual HDF5 files may vary depending on who produced the data. In this
+case, the Wavelength and reflectance data within the file are both datasets.
+However the spatial information is contained within a group. Data downloaded from
+another organization like NASA, may look different. This is why it's important to
 explore the data before diving into using it!
 </div>
 
 We can use the `h5readAttributes` function to read and extract metadata from the 
-HDF5 file. Let's start by reading in the spatial information.
-
-
-```{r read-spatial-attributes}
+HDF5 file. Let's start by learning about the wavelengths described within this file.
 
-# get spatialInfo using the h5readAttributes function 
-spInfo <- h5readAttributes(f,"spatialInfo")
-
-# get attributes for the Reflectance dataset
-reflInfo <- h5readAttributes(f,"Reflectance")
+```{r read-band-wavelength-attributes }
 
+# get information about the wavelengths of this dataset
+wavelengthInfo <- h5readAttributes(f,"/SJER/Reflectance/Metadata/Spectral_Data/Wavelength")
+wavelengthInfo
 
 ```
 
-Next, let's read in the wavelength center associated with each band in the HDF5 
-file. 
+Next, we can use the `h5read` function to read the data contained within the
+HDF5 file. Let's read in the wavelengths of the band centers:
 
 ```{r read-band-wavelengths }
-
 # read in the wavelength information from the HDF5 file
-wavelengths<- h5read(f,"wavelength")
+wavelengths <- h5read(f,"/SJER/Reflectance/Metadata/Spectral_Data/Wavelength")
+head(wavelengths)
+tail(wavelengths)
 
 ```
 
-Which wavelength is band 19 associated with? (Hint: look at the wavelengths 
-vector that we just imported and check out the data located at index 19 - 
-`wavelengths[19]`).
+Which wavelength is band 6 associated with? (Hint: look at the wavelengths 
```

Comment:
I like putting a hard return and an extra row before hints. This allows the person to consider the question, before getting the hint, if they want to challenge themselves.

Reply:
Good point - I have made these changes.